### ISSUE TITLE
CreateNewTestAccount SDK method for TestNet.

### ISSUE THREAD

**Issue by rfustino** [algorand/go-algorand#2895]
## Problem

Ease of use  for our developers is the problem being solved, when funding new TestNet accounts at the time they are created.

## Solution

Provide method call , similar to Reach, on creating a new test account and populating it with funds. For example,  CreateNewTestAccount(10) would create a new account and fund it with 10 TestNet Algos. Reach provides this function only for Private Network. The function would need to detect where it is running so it only works on TestNet/BetaNet.  

## Dependencies

This issue should be resolved on rate limiting to dispensers. https://github.com/algorand/go-algorand/issues/2892

## Urgency

If urgency is related to providing a better developer experience, than this would facilitate that effort. Dev-rel is creating new replit content and using the best practice to remove mnemonic phrases from sample code. So the sequence for every replit is to create an account, prompt the user to fund the account and press any key to continue. This is cumbersome at best and we can  facilitate a better ease of use experience here. Example: replit: https://replit.com/@Algorand/Getting-Started-with-JavaScript?v=1 . 

In other words, the ideal experience would be to have the dev hit the run button on the replits and not have to do anything but watch the code execute. At the moment, the developer needs to do this sequence... 

- see the prompt 
- copy the url from the prompt 
- paste to a browser in another tab 
- hit the captcha 
- dispense 
- wait 5 seconds (Which can be replaced by a wait for confirmation call in the code), 
- navigate back to the tab with the app running 
- hit enter to continue

A much better experience, especially for a new developer, is to simply hit the run button on these replits. In addition to replits, this would also help ease of use our content/demo examples on the dev portal.
